users/serializers.py

```diff
@@ -25,6 +27,7 @@ class Meta(UserPublicSerializer.Meta):
             "description",
             "date_of_birth",
             "profile_photo",
+            "email",
```

Comment:
Let's add readonly `id` field to all serializers

Reply:
Done